pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -216,6 +216,9 @@ def work(self):
                                         if len(pokemon_to_transfer) == 0:
                                             raise RuntimeError(
                                                 'Trying to transfer 0 pokemons!')
+                                        # Add slight delay between capture & candy transfer #774
+                                        logger.log("Waiting briefly before transferring pokemon")
+                                        sleep(3)
```

Comment:
This isn't `time.sleep(3)` it's `sleep(3)` using the function in human_behavior.py. 10 sample outputs using this function as implemented:

>2.67440283374
>2.43061394227
>3.41717271461
>3.16960045279
>3.60687241192
>3.77872126265
>2.84796236007
>2.64138294609
>3.11884116248
>2.61486735064